### PR TITLE
Sync Android and JVM packages to latest

### PR DESCRIPTION
**Pull request by sugarmanz**
### Change Type (required)
Indicate the type of change your pull request is:

- [ ] `patch`
- [x] `minor`
- [ ] `major`

### CODE REVIEW

File: android/player/src/main/res/values/tags.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item name="view_hydrated" type="bool" />
```
Comment (brocollie08) [inline android/player/src/main/res/values/tags.xml:3]: was this for the automation tests?

File: jvm/BUILD [hunk 2]
```diff
@@ -49,6 +49,11 @@ kt_kotlinc_options(
 
 ktlint_config(
     name = "lint_config",
-    editorconfig = "//:.editorconfig",
+    #    editorconfig = "//:.editorconfig",
```
Comment (brocollie08) [inline jvm/BUILD:52]: intentional?

File: WORKSPACE [hunk 3]
```diff
@@ -119,6 +118,18 @@ load("@robolectric//bazel:robolectric.bzl", "robolectric_repositories")
 
 robolectric_repositories()
 
+ANDROIDX_TEST_VERSION = "1.4.2"
```
[the remark this answers is not in the document]
Comment (sugarmanz) [reply WORKSPACE:121]: Nope! Removed, will push soon

File: android/demo/BUILD [hunk 4]
```diff
@@ -86,6 +87,8 @@ sh_test(
     data = [
```
[the remark this answers is not in the document]
Comment (sugarmanz) [reply android/demo/BUILD:87]: `deps` is for dependencies that are required at build time, `data` is for dependencies that are required at run time, which all of these aren't needed until the shell script actually executes.

File: .circleci/config.yml [hunk 5]
```diff
@@ -249,6 +249,19 @@ jobs:
 
       - run: bazel test --define=APPLITOOLS_API_KEY=${APPLITOOLS_API_KEY} --define=APPLITOOLS_BATCH_ID=${CIRCLE_SHA1} --define=APPLITOOLS_PR_NUMBER=${CIRCLE_PULL_REQUEST##*/} --config=ci -- //android/demo:android_instrumentation_test
 
+      - run:
```
[the remark this answers is not in the document]
Comment (sugarmanz) [reply .circleci/config.yml:252]: This is actually to let `circleci` know that we ran tests and what the results of those tests are

File: android/demo/scripts/androidtest.sh [hunk 6]
```diff
@@ -2,7 +2,73 @@
...
+adb logcat -c &&
+python - <<END
+import os
+import re
+import subprocess as sp
+import sys
+import threading
+import time
+done = False
+def update():
```
[the remark this answers is not in the document]
Comment (sugarmanz) [reply android/demo/scripts/androidtest.sh:44]: Yep - this was copy pastad from somewhere, I'll make it more legible and add a link to where I got it from in the first place

File: android/player/src/main/res/values/tags.xml (hunk 1 above)
Comment (sugarmanz) [reply android/player/src/main/res/values/tags.xml:3]: Yeah, it's the only way we currently have to determine if an asset has been completely hydrated and ready for use

File: jvm/BUILD (hunk 2 above)
Comment (sugarmanz) [reply jvm/BUILD:52]: Yep, will remove the comment in my next push

File: android/demo/BUILD (hunk 4 above)
[the remark this answers is not in the document]
Comment (sugarmanz) [reply android/demo/BUILD:87]: Rekt:
CleanShot 2023-12-04 at 19 49 12

**Pull request merged**